lib/config-util.js

```diff
@@ -219,6 +219,17 @@ Config.buildYargs = function (cwd) {
       type: 'boolean',
       global: false
     })
+    .option('clean', {
+      describe: 'should the .nyc_output folder be cleaned before executing tests',
+      default: true,
```

Comment:
this allows us to keep the folder around between two runs of `nyc` -- which allows reports to be combined from mocha and tap -- as we start migrating a few tests over.